compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -243,7 +237,7 @@ pub fn target_features(sess: &Session, allow_unstable: bool) -> Vec<Symbol> {
     // (see https://reviews.llvm.org/D110413). This unstable target feature is intended for use
     // by compiler-builtins, to export the builtins with the expected, LLVM-version-dependent ABI.
     // The target feature can be dropped once we no longer support older LLVM versions.
-    if sess.is_nightly_build() && get_version() >= (14, 0, 0) {
+    if sess.is_nightly_build() {
```

Comment:
I guess this could happen in a followup, but we should update compiler-builtins to stop checking this feature and remove it here.

Reply:
Now or later, either is fine with me, but here's that update:
https://github.com/rust-lang/compiler-builtins/pull/514

Reply:
Done!